### PR TITLE
Solution attempt

### PR DESCRIPTION
**Pull request by Likh-Alex**

### CODE REVIEW

File: src/main/java/core/basesyntax/UserService.java [hunk 1]
```diff
@@ -1,8 +1,17 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/UserService.java]: remove empty line

File: src/main/java/core/basesyntax/exception/UserNotFoundException.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.exception;
+
+public class UserNotFoundException extends RuntimeException {
+    public UserNotFoundException(String message) {
+        super(message);
+    }
+
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/exception/UserNotFoundException.java]: and here too

**Review by Sofasmile — APPROVED** [review]
good

File: src/main/java/core/basesyntax/UserService.java (hunk 1 above)
Comment (Likh-Alex) [reply src/main/java/core/basesyntax/UserService.java]: Done, thank you

File: src/main/java/core/basesyntax/exception/UserNotFoundException.java (hunk 2 above)
Comment (Likh-Alex) [reply src/main/java/core/basesyntax/exception/UserNotFoundException.java]: Done here too, thank you

**Pull request closed without merging**